### ISSUE TITLE
Contrast ratio of selected option from list controls is 1.2:1 which is less than 3:1

### ISSUE THREAD

**Issue by adam-msft** [microsoft/fluentui#16122]
User Impact:
Minimum color contrast ratio of 3:1 is not provided between focus and its background for a selected option within the dropdown control (https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown). As a result, it would be difficult for people with moderately low vision to identify the selected option from list controls present on the dialog.
 
Test Environment:
OS: Windows 10 version 2004(Build 20197.1000) 
Browser: New Edge (Version 84.0.522.63) 
Tool: Color Contrast Analyzer (Version 3.1.1)

Steps to Reproduce:
1. Open URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown on New Edge (Chromium) browser.
2. Navigate to a dropdown and activate it.
3. Turn on Color Contrast Analyzer tool and select the foreground and background color of selected list item
4. Observe that the contrast ratio is equal to or greater than 3:1 or not.

Actual Result:
The contrast ratio of selected option from list controls is 1.2:1 which is less than 3:1.

Expected Result:
The color contrast ratio of selected option from list controls should be equal to or greater than 3:1.

**Comment by JustSlone** [comment]
@adam-msft thanks for filing this!

We'll reach out to our design team to figure out what the updated color should be.

**Comment by PoojaNamde** [comment]
We have discussed this bug with Accessibility SME Scott (scott.ohara@microsoft.com) and this needs to be fixed by adding a border that passes the contrast.  This could cause issues for persons with low vision especially if they're using high zoom levels and have smaller viewports where the element and the dropdown options are not completely visible. Also, if there is a long list and the user wants to know where the focus is, they need to look at the option showing up on top and then look for it in the list.

If you have any concerns, please get in touch with Scott
